src/models/multiChart.js

```diff
@@ -90,8 +91,8 @@ nv.models.multiChart = function() {
                     })
                 });
 
-            x   .domain(d3.extent(d3.merge(series1.concat(series2)), function(d) { return d.x } ))
-                .range([0, availableWidth]);
+            x   .domain(d3.extent(d3.merge(series1.concat(series2)), function(d) {
+                return xAxisDates ? new Date(d.x) : d.x } )).range([0, availableWidth]);
```

Comment:
Hmm... Seems like this should be using chart.x()(d) to get the value... Then you could just override the x getter option to return a date....